### ISSUE TITLE
[Deque Analysis] Submission Form Collection Selector has insufficient color contrast

### ISSUE THREAD

**Issue by tdonohue** [DSpace/dspace-angular#1158]
## Deque Analysis Summary

On the Submission form, the Collection selector (at the top of the page) has several color contrast issues

https://demo7.dspace.org/submit

1. "Collection" label: "Element has insufficient color contrast of 3.77 (foreground color: `#777777`, background color: `#e9ecef`). Contrast must be 4.5 or more."
     * Deque issue ticket IDs: 469505
2. In dropdown, placeholder text "Search for a collection": "Placeholder text lacks 4.5:1 contrast ratio. Foreground: `#9F9F9F`, Background: `#FFFFFF`, The contrast ratio is: 2.6:1"
     * Deque issue ticket IDs: 469775
3. In dropdown, Community names (appearing above Collection names): "Button text content lacks 4.5:1 contrast ratio on hover and on focus. Default state: Foreground: `#777777`, Background: `#FFFFFF`, The contrast ratio is: just below 4.5:1 (4.478:1) On hover and on focus: Foreground: `#777777`, Background: `#E8EBF3`, The contrast ratio is: 3.8:1"
     * Deque issue ticket IDs: 469777
4. When focused, Collection dropdown does not have at least a 3 to 1 luminosity contrast ratio with either the inner or outer adjacent background. Focus indicator color: `#A1A8AF`, Background: `#FFFFFF`, The contrast ratio is: 2.4:1
    * Deque issue ticket IDs: 469778

## Recommended Fix

Minimally, the text needs to be made darker (or use a different style).  In all these cases, the text color `#777777` is insufficient.  For a white background it'd need to be `#757575` at a minimum.  For darker backgrounds it must be darker.  Use this tool to select a better foreground/background color: https://dequeuniversity.com/rules/axe/4.1/color-contrast?application=axeAPI

## More Information / Tools

* As necessary, see mentioned Issue IDs in our assessment results (requires login): https://axeauditor.dequecloud.com/test-run/0856438a-a19a-11eb-bc31-b7d5be387c86/issues
* Consider using [Deque's free Chrome plugin](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) to check your work
* Deque's color contrast tool: https://dequeuniversity.com/rules/axe/4.1/color-contrast?application=axeAPI

**Comment by tdonohue** [comment]
@atarix83 : Assigning this to you for your analysis as it has to do with the Collection selector on the Submission form.

**Comment by atarix83** [comment]
@tdonohue scanning the page with axe DevTools doesn't report the errors indicated there, is it possible they have been already resolved with other accessibility issue ?

**Comment by tdonohue** [comment]
@atarix83 : I think you are correct as I'm not seeing those color contrast issues any longer either.  This looks like this may have been fixed by #1241 which fixed similar color contrast issues in other areas of the application.

Closing this as fixed.
